File: src/main/java/commands/slashadapters/adapters/ColorAdapter.java
package commands.slashadapters.adapters;

import commands.runnables.aitoyscategory.ColorCommand;
import commands.slashadapters.Slash;
import commands.slashadapters.SlashMeta;
import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;

@Slash(command = ColorCommand.class)
public class ColorAdapter extends AIAdapterAbstract {

    @Override
    public SlashMeta process(SlashCommandEvent event) {
        return new SlashMeta(ColorCommand.class, collectArgs(event));
    }

}

File: src/main/java/commands/slashadapters/adapters/ColorAdapter.kt
package commands.slashadapters.adapters

import commands.runnables.aitoyscategory.ColorCommand
import commands.slashadapters.Slash
import commands.slashadapters.SlashMeta
import mysql.hibernate.entity.guild.GuildEntity
import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent

@Slash(command = ColorCommand::class)
class ColorAdapter : AIAdapterAbstract() {

    override fun process(event: SlashCommandInteractionEvent, guildEntity: GuildEntity): SlashMeta {
        return SlashMeta(ColorCommand::class.java, collectArgs(event))
    }

}